src/nvim/globals.h

```diff
@@ -117,12 +117,8 @@
 #define DFLT_COLS       80              /* default value for 'columns' */
 #define DFLT_ROWS       24              /* default value for 'lines' */
 
-EXTERN long Rows                        /* nr of rows in the screen */
-#ifdef DO_INIT
-  = DFLT_ROWS
-#endif
-;
-EXTERN long Columns INIT(= DFLT_COLS);         /* nr of columns in the screen */
+EXTERN NumOpt Rows INIT(= DFLT_ROWS);     // nr of rows in the screen
```

Comment:
Apparently not only me was thinking that `///<` is member documentation. I do not remember `///<` used for non-members, and `ag` found only seven of them.

(Note that I added this comment before adding https://github.com/neovim/neovim/pull/3914/files#r49841130: I simply did not think `///<` is applicable.)